src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -45,6 +45,39 @@ public async Task SendAsync_UserAgent_CorrectlyWritten()
             });
         }
 
+        [Fact]
+        public async Task SendAsync_LargeHeaders_CorrectlyWritten()
```

Comment:
We should add a comment here explaining that this is intentionally larger than 16K in total because that's the limit that will trigger a CONTINUATION frame in HTTP2.

Reply:
Added.